Foundation/NSURLSession/EasyHandle.swift

```diff
@@ -191,22 +188,10 @@ extension _EasyHandle {
     }
     /// Wait for pipelining/multiplexing
     /// - SeeAlso: https://curl.haxx.se/libcurl/c/CURLOPT_PIPEWAIT.html
-    //func set(waitForPipeliningAndMultiplexing flag: Bool) {
-    //    try! CFURLSession_easy_setopt_long(rawHandle, CFURLSessionOptionPIPEWAIT, flag ? 1 : 0).asError()
-    //}
     
     //TODO: The public API does not allow us to use CFURLSessionOptionSTREAM_DEPENDS / CFURLSessionOptionSTREAM_DEPENDS_E
     // Might be good to add support for it, though.
     
-    /// set numerical stream weight
-    /// - Parameter weight: values are clamped to lie between 0 and 1
-    /// - SeeAlso: https://curl.haxx.se/libcurl/c/CURLOPT_STREAM_WEIGHT.html
-    /// - SeeAlso: http://httpwg.org/specs/rfc7540.html#StreamPriority
-    //func set(streamWeight weight: Float) {
-    //    // Scale and clamp such that the range 0->1 ends up 1->256
-    //    let w = 1 + max(0, min(255, Int(round(weight * 255))))
-    //    try! CFURLSession_easy_setopt_long(rawHandle, CFURLSessionOptionPIPEWAIT, w).asError()
-    //}
```

Comment:
May make sense to document what the CURLOPT_PIPEWAIT option helps achieve here ..